fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/models/FedoraResource.java

```diff
@@ -257,20 +251,12 @@ void replaceProperties(final IdentifierConverter<Resource, FedoraResource> idTra
     String getEtagValue();
 
     /**
-     * Enable versioning
-     */
-    void enableVersioning();
-
-    /**
-     * Disable versioning
-     */
-    void disableVersioning();
-
-    /**
-     * Check if a resource is versioned
-     * @return whether the resource is versioned
+     * Check if a resource is an original resource
+     * (ie versionable, as opposed to non-versionable resources
+     * like mementos, timemaps, and acls).
```

Comment:
is there ever a case for acls to be versioned?

Reply:
No. Not at this time.